virttest/qemu_devices/qcontainer.py

```diff
@@ -609,6 +616,13 @@ def has_device(self, device):
         return bool(re.search(r'name "%s"|alias "%s"' % (device, device),
                               self.__device_help))
 
+    def has_object(self, obj):
+        """
+        :param obj: Desired object string, e.g. 'sev-guest'
+        :return: True if the object is supported by qemu, or False
+        """
+        return obj in self.__object_help
```

Comment:
# /usr/libexec/qemu-kvm -object \?
List of user creatable objects:
  authz-list
  authz-list-file
  authz-simple
  colo-compare
  cryptodev-backend
  cryptodev-backend-builtin

One line one backend